lib/hammer_cli_foreman/audit.rb

```diff
@@ -0,0 +1,71 @@
+module HammerCLIForeman
+
+  class Audit < HammerCLIForeman::Command
+
+    resource :audits
+
+    def request_params
+      params = super
+      params['search'] ||= 'time >= "1 month ago"'
```

Comment:
somehow this does not seem to apply as a default value, it still lists all audits for me

`hammer audit list --search 'time >= "1 month ago"'` when I explicitly set the search ir works, I added pry here and it was not triggered so I think @tstrachota had a valid concern above.

Reply:
moving to ListCommand works, but also you must add params as last line of this method, otherwise it returns the search string only